src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -121,6 +121,8 @@ public static class ScriptConstants
 
         public const string AzureWebJobsHostsContainerName = "azure-webjobs-hosts";
 
+        public const string InstalledDependencyInfoFileName = "DependencyInfo.json";
```

Comment:
Same here. This is specific to Powershell language worker. Please move this to the worker